app/models/legislator.rb

```diff
@@ -44,13 +44,13 @@ class Legislator < ActiveRecord::Base
   validates :state,    presence: true, if: :senator?
   validates :district, absence:  true, if: :senator?
 
-  scope :senate,       -> { where(chamber: 'senate') }
-  scope :house,        -> { where(chamber: 'house') }
-  scope :eligible,     -> { where('term_end < ?', 2.years.from_now) }
-  scope :targeted,     -> { joins(:campaigns).merge(Campaign.active) }
-  scope :top_priority, -> { targeted.merge(Target.top_priority) }
-  scope :unconvinced,  -> { where(with_us: false) }
-  scope :with_us,      -> { where(with_us: true) }
+  scope :senate,      -> { where(chamber: 'senate') }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.